app/src/main/res/values/strings.xml

```diff
@@ -559,10 +559,10 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="quest_religion_for_wayside_shrine_title">What religion is represented at this shrine?</string>
     <string name="quest_parking_fee_title">Do you have to pay to park here?</string>
     <string name="quest_surface_value_unhewn_cobblestone">Unhewn cobblestone</string>
-    <string name="quest_parking_access_title">Is it restricted who may park here?</string>
-    <string name="quest_parking_access_private">It is private</string>
-    <string name="quest_parking_access_yes">It is open to the public (with or without fee)</string>
-    <string name="quest_parking_access_customers">It is only for customers</string>
+    <string name="quest_parking_access_title2">Who may park here? Parking may be free or paid.</string>
+    <string name="quest_parking_access_yes2">Open to general public</string>
+    <string name="quest_parking_access_private2">Not open to general public</string>
+    <string name="quest_parking_access_customers2">Open only to people visiting a specific place (e.g. customers)</string>
```

Comment:
One or both of these two should include the keyword *reserved*.

Reply:
> I am also not sure I fully understand the distinction between private and customers. Perhaps you could share a more technical definition, and I can try to come up with a good phrasing for that?

https://wiki.openstreetmap.org/wiki/Tag:access%3Dcustomers

https://wiki.openstreetmap.org/wiki/Tag:access%3Dprivate

`amenity=parking access=private` - "only for residents"

`amenity=parking access=customers` - "open only to people who are shopping at specific location (or meeting lawyer or picking up children from school or leaving shoes for repair)"

Reply:
"private" can also mean that they are rented, doesn't need to be residents. But there will be a sign. In any case, not relevant for the public.

Reply:
> private parking may have public access (say, paid parking near airport on private grounds)

I think you mean "privately-owned parking", otherwise this seems like a contradiction. So, to clarify, paid parking near an airport could be `access=public` _if_ you are allowed to park there even if you are not flying anywhere.

<details><summary>By the way, this is also a good example of much the most confusing situation, which I mentioned above.</summary>

- On one hand, the parking is clearly *primarily* for people using the airport.
- On the other, it *may* be available for public use.

*Sometimes* "for customers only" is signed, which makes it clear. When there is no sign, I usually skip the question because I'm not sure what the laws are.

Aside: while I don't feel comfortable tagging these places as `access=public`, I would usually not hesitate to park there myself, even if I was not a customer, since that restriction is rarely enforced, especially when not signed.
</details>

> publicly owned parking may have private access (parking where just president of city is allowed to park)

So, correct me if I am wrong— it seems like the dividing line is:

- `access=private` → It is reserved for some specific persons
- `access=customers` → It is open to any person, but only under some conditions (e.g. intent to shop at a certain place)

Reply:
Assuming ^that understanding is correct, I think we should also swap the order in the form, so it's

- public
- customers
- private

That way it's in order from least to most restrictive.

Reply:
> That way it's in order from least to most restrictive.

Done

> access=customers → It is open to any person, but only under some conditions (e.g. intent to shop at a certain place)

Yes, though I would also it describe as in general possible to general public (so "only for president of a country" would be `access=private`)

Reply:
> "only for president of a country" would be `access=private`

Right, because one can't simply decide to become the president. Same thing would apply to "only for employees" → `access=private`; a random member of the public can't simply decide to become an employee; it requires an agreement *in advance*.

Okay, so I think I understand now. I still need to think more about how to phrase this clearly and succinctly.

Reply:
Though note that for example school parking that can be used by parents picking up children I would mark as `access=customers` Despite that you cannot decide one day to have child and go sign up for specific school.

>I still need to think more about how to phrase this clearly and succinctly.

Let me know if you have any better ideas! I am not entirely happy about wording here but existing has bigger problems and this seems to be a clear improvement to me.

But likely it can be even better.

Reply:
I think that's a great edge case. You could argue that a friend of the family might pick the child up one day, but I think that's still similar to `private` (more like a driveway— the parent would give permission in advance).

I can think of other reasons to tag `customers`. In my area, at least, schools are often used as voting buildings (in the gymnaseum or cafeteria). Also, I'm pretty sure some of the facilities (pool, tennis court, track, soccer field) are open to other people at least some of the time.

But, as a thought experiment, let's say they weren't and the only use for the school parking lot is parking for teachers or a parent picking up their child. Would you still tag it `access=customers`? If so, why?

Reply:
> Would you still tag it access=customers? If so, why?

Mostly because it fits well "in general not allowed to park there, but general public using this place would be allowed to use it".

Though I admit that I would mark "employee-only" parking near factory as `access=private`

It may be worth discussing on tagging mailing list or OSM wiki and maybe inventing new values. (if anyone wants to do this: feel free, I am unlikely to do this)

Reply:
I am also unlikely :D. From the [wiki `key:access` page](https://wiki.openstreetmap.org/wiki/Key:access), it looks like this tag has a history of being confusing (many deprecated values).

In any case, I think I understand the tag better now, and have made updated suggestions below, so I'll resolve this.